bLactam_MIC_Rscripts/Build_PBP_AA_tableR3.2.2.R

```diff
@@ -11,13 +11,14 @@ Build_PBP_AA_table<- function(datafolder)
 
   setwd(datafolder)
 
-  libpath="/scicomp/groups/OID/NCIRD/DBD/RDB/Strep_Lab/External/share/PBP_AA_to_MIC/Rlib"
+  #libpath="/scicomp/groups/OID/NCIRD/DBD/RDB/Strep_Lab/External/share/PBP_AA_to_MIC/Rlib"
```

Comment:
This is done as part of the installation, using the standard R environment variables as opposed to code this in the functions.  It's more manageable that way.